dspace/config/spring/api/core-services.xml

```diff
@@ -45,7 +45,17 @@
     <bean class="org.dspace.content.BundleServiceImpl"/>
     <bean class="org.dspace.content.CommunityServiceImpl"/>
     <bean class="org.dspace.content.InstallItemServiceImpl"/>
-    <bean class="org.dspace.content.ItemServiceImpl"/>
+    <bean class="org.dspace.content.ItemServiceImpl">
+        <property name="nonCopyableMetadataFields">
+            <set>
+                <value>dc.date.accessioned</value>
+                <value>dspace.entity.type</value>
```

Comment:
This field arguably *should* be copied, as you'd want to ensure an existing Entity is copied into the same type of object (e.g. you wouldn't want to copy a Person Entity into a plain Item).